extensions/kafka/src/main/java/io/deephaven/kafka/KafkaTools.java

```diff
@@ -1603,15 +1603,50 @@ public Function<TopicPartition, KafkaRecordConsumer> visit(@NotNull final PerPar
      *        {@link TableDefinition}. See {@link StreamConsumerRegistrarProvider#single(SingleConsumerRegistrar)
      *        single} and {@link StreamConsumerRegistrarProvider#perPartition(PerPartitionConsumerRegistrar)
      *        per-partition}.
+     * @return the newly created KafkaIngester, the caller must call start on the returned object to begin processing
+     *         messages
      */
-    public static void consume(
+    public static KafkaIngester consume(
             @NotNull final Properties kafkaProperties,
             @NotNull final String topic,
             @NotNull final IntPredicate partitionFilter,
             @NotNull final IntToLongFunction partitionToInitialOffset,
             @NotNull final Consume.KeyOrValueSpec keySpec,
             @NotNull final Consume.KeyOrValueSpec valueSpec,
             @NotNull final KafkaTools.StreamConsumerRegistrarProvider streamConsumerRegistrarProvider) {
+        return consume(kafkaProperties, topic, partitionFilter,
+                new KafkaIngester.IntToLongFunctionAdapter(partitionToInitialOffset), keySpec, valueSpec,
+                streamConsumerRegistrarProvider);
+    }
+
+    /**
+     * Consume from Kafka to a {@link StreamConsumer} supplied by {@code streamConsumerRegistrar}.
```

Comment:
Can you correct the one you copied from, too?
```suggestion
     * Consume from Kafka to {@link StreamConsumer stream consumers} supplied by {@code streamConsumerRegistrar}.
```